src/script/run-make.sh

```diff
@@ -88,7 +88,11 @@ function prepare() {
     fi
 
     if test -f ./install-deps.sh ; then
-	    export WITH_SEASTAR=1
+            if [ "$WITHOUT_SEASTAR" ] ; then
```

Comment:
can we use `WITH_SEASTAR` instead of introducing yet another variable? as `install-deps.sh` also check for this `WITH_SEASTAR`.

Reply:
Any solution that allows the user to trigger a non-Seastar `run-make-check.sh` run will be good for me.

If you want to use `WITH_SEASTAR`, we could simply remove this `export WITH_SEASTAR=1` line from `run-make-check.sh`. However, this would change the default, so we would need to modify Jenkins to set this variable on the make check runs.

Reply:
https://github.com/ceph/ceph-build/pull/1515

Reply:
#33723